rootio/wbuffer_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package rootio
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestWbuffer(t *testing.T) {
+	data := make([]byte, 20)
+	wbuf := NewWBuffer(data, nil, 0, nil)
+	var i bool
```

Comment:
these 2 lines could just be written as:
```go
i := true
```

actually, please rename that variable as `want` instead of `i`.